src/test/java/StringManipulationTest.java

```diff
@@ -27,18 +29,23 @@ public void testCount1() {
     }
 
     @Test
```

Comment:
Based on the comment I made in the StringManipulation.java, I would say change this test so it counts two words here.